sys-fs/cryfs/cryfs-9999.ebuild

```diff
@@ -54,15 +64,15 @@ src_test() {
 	local TMPDIR="${T}"
 	addread /dev/fuse
 	addwrite /dev/fuse
-	local tests_failed=""
+	local tests_failed=()
 
 	for i in gitversion cpp-utils parallelaccessstore blockstore blobstore fspp cryfs cryfs-cli ; do
-		${BUILD_DIR}/test/${i}/${i}-test || tests_failed+="${i} "
+		"${BUILD_DIR}"/test/${i}/${i}-test || tests_failed+=( "${i}" )
 	done
 
 	adddeny /dev/fuse
 
-	if [ -n "$tests_failed" ] ; then
+	if [[ -n ${tests_failed[@]} ]] ; then
 		eerror "The following tests failed:"
 		eerror "$tests_failed"
```

Comment:
Whoops, forgot that when I was changing the other stuff to make it use a bash array instead of a single string. I've gotten in the habit of avoiding bashisms in shell scripts where possible, hence this and the single bracket test.